lib/api/core/entrypoints/embedded/index.js

```diff
@@ -379,7 +413,9 @@ class EmbeddedEntryPoint extends EntryPoint {
           queryString += '&';
         }
 
-        queryString += `${k}=${typeof val === 'object' ? JSON.stringify(val) : val}`;
+        queryString += `${k}=${typeof val === 'object'
```

Comment:
I guess it works, but I'm not sure that wrapping at 80 chars inside a template litteral is a good idea